app/controllers/skins_controller.rb

```diff
@@ -30,13 +30,13 @@ def index
       end
     else
       if is_work_skin
-        @skins = WorkSkin.approved_skins.sort_by_recent_featured
+        @skins = WorkSkin.approved_skins
```

Comment:
I think maybe you should change this to another scope like `sort_by_recent`, instead of deleting the scope altogether. The `sort_by_recent_featured` scope ordered first by `featured`, then by `updated_at`. So it'd show all the featured skins, ordered by last update, and then all of the unfeatured skins (also ordered by last update). By deleting this, you're deleting both ordering constraints, which means that the skins will be arranged in whatever order the database defaults to. (Which I think will be increasing order by ID. Oldest first, instead of recently updated first.)

I suspect this is what's causing the automated test error in `features/other_b/skin_public.feature` (which doesn't appear to be intermittent).